packages/playground/wordpress/src/boot.ts

```diff
@@ -91,6 +92,19 @@ export interface BootOptions {
 	 * given request URI.
 	 */
 	getFileNotFoundAction?: FileNotFoundGetActionCallback;
+
+	/**
+	 * - `internal-store`: Persist cookies from reponses in an internal store and
+	 * includen them in following requests. This is a behavior mostly needed when
+	 * using Playground in web.
+	 *
+	 * - `pass-through`: Avoid persisting cookies internally and let them pass
+	 * through the requests and back to the caller from responses. This is the
+	 * common behavior in requests handled by a server.
```

Comment:
Let's expand this a bit:

```suggestion
	 * - `internal-store`: Persist cookies from reponses in an internal store and
	 * includes them in following requests. This behavior is useful in the Playground
	 * web app because it allows multiple sites to set cookies on the same domain 
	 * without running into conflicts. Each site gets a separate "namespace". The
	 * downside is that all cookies are global and you cannot have two users 
	 * simultaneously logged in.
	 *
	 * - `pass-through`: Typical server behavior. All cookies are passed back to the
	 * client via a HTTP response. This behavior is useful in when Playground is
	 * running on the backend and can be requested by multiple browsers. This enables
	 * each browser to get its own cookies, which means two users may be simultaneously 
	 * logged in to their accounts.
```

Reply:
I saw it's duplicated in another type. Is there a way to move that documentation to CookieStrategy type and still see it in the autocompletion tooltip when working with the BootOptions type?

Reply:
> Is there a way to move that documentation to CookieStrategy type and still see it in the autocompletion tooltip when working with the BootOptions type?

Good question. I had the same concern but I couldn't find a way to reuse the documentation. In any case, I agree we could avoid duplication if we move it to the type.